skimage/morphology/__init__.py

```diff
@@ -1,3 +1,11 @@
+"""Collection of broad set of operations that process images based on shapes.
```

Comment:
Found a minor grammar issue after approving. It should be "Collection of **a** broad set". ;) 

What do you think about making this a bit more concise? "Collection" and "set" seems redundant.
```suggestion
"""Utilities that process images based on shapes.
```